pkg/operator/clustermemberremovalcontroller/clustermemberremovalcontroller_test.go

```diff
@@ -696,9 +739,15 @@ func TestAttemptToScaleDown(t *testing.T) {
 				configMapListerForTargetNamespace: configMapTargetNSLister,
 			}
 			err = target.attemptToScaleDown(context.TODO(), eventRecorder)
-			if err != nil {
+			if err == nil && scenario.expectedError != nil {
+				t.Fatal("expected to get an error from attemptToScaleDown method")
```

Comment:
Nit: show the expected error
```suggestion
				t.Fatal("expected to get an error from attemptToScaleDown method, expected = %v", scenario.expectedError)
```